tests/migration/online_migration/pre_migration.pm

```diff
@@ -61,6 +62,9 @@ sub run {
     remove_kgraft_patch if is_sle('<15');
     # create btrfs subvolume for aarch64 before migration
     create_btrfs_subvolume() if (is_aarch64);
+    # We need to close gnome notification banner before migration.
+    select_console 'user-console';
+    turn_off_gnome_show_banner if check_var('DESKTOP', 'gnome');
```

Comment:
Is this for specific arch? If so please add the corresponding condition.

Reply:
> Is this for specific arch? If so please add the corresponding condition.

No,  now both aarch and ppc64le will hit this issue.

Reply:
So this is still on specific arches. limit the fix for aarch64 and ppc64le except you can prove it is needed for other arches also.

Reply:
Changed.